composer.json

```diff
@@ -70,26 +64,23 @@
         ],
         "test:mutation": [
             "Composer\\Config::disableProcessTimeout",
-            "tools\/infection\/vendor\/bin\/infection -j2"
+            "tools\/infection\/vendor\/bin\/infection --threads=4"
```

Comment:
` --threads=max` is probably the best

Reply:
I tried but it was slower than `--threads=4`

Reply:
TIL 😅